MSAL/src/requests/MSALRequestParameters.h

```diff
@@ -49,13 +51,18 @@
 @property MSALTelemetryApiId apiId;
 @property NSDictionary<NSString *, NSString *> *sliceParameters;
 
+@property MSALWebviewType webviewType;
+
+@property WKWebView *customWebview;
+
 #pragma mark MSALRequestContext properties
 @property NSUUID *correlationId;
 @property NSString *logComponent;
 @property NSString *telemetryRequestId;
 @property NSURLSession *urlSession;
 
 @property (retain, nonatomic, readonly) MSIDConfiguration *msidConfiguration;
+@property (retain, nonatomic) MSIDOauth2Factory *msidOAuthFactory;
```

Comment:
it's weird that those are the only nonatomic properties in here...

Reply:
I've removed a retain and nonatomic here, and also created an issue #299 to revisit the whole class.